nuxt/cypress/e2e/account.cy.ts

```diff
@@ -0,0 +1,17 @@
+describe('account test', () => {
+  beforeEach(() => {
+    cy.visit('/account')
+  })
+
+  // This is currently broken, due to nuxt3 redirects not working properly
+  // Should be fixed in nuxt v3.0.0-rc.12
```

Comment:
We use the RC12 *edge* build already, but most probably the `redirect` `pageMeta` needs to be used instead of `navigateTo`. Just fyi. It won't be fixed in RC12 automatically.